client/client-multi/client-c/tools/kaa_encryption/rsa_key_gen/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(kaa_rsa_keygen C)
+
+add_library(rsa_keygen kaa_rsa_key_gen.c)
+include_directories(rsa_keygen ${CMAKE_CURRENT_LIST_DIR})
+
+add_subdirectory("../../../thirdparty/mbedtls" kaa_mbedtls)
```

Comment:
Let's use `KAA_DIR` or `KAA_ROOT` or whatever defined in the root `CMakeLists.txt` instead of `../../../`